src/components/Aside/Aside.tsx

```diff
@@ -0,0 +1,108 @@
+import { useEffect, useRef } from "react";
+import styles from "./Aside.module.css";
+import Seed from "../../assets/Seed.png";
+import Sprout from "../../assets/Sprout.png";
+import YoungTree from "../../assets/YoungTree.png";
+import LargeTree from "../../assets/LargeTree.png";
+import Github from "../../assets/Github.png";
+import { Grade } from "../../api/services/common/types.ts";
+
+interface AsideProps {
+  githubUsername: string;
+  easyProgress: string;
+  mediumProgress: string;
+  hardProgress: string;
+  solvedTasks: number;
+  totalTasks: number;
+  profile_url: string;
+  cohort: number;
+  grade: Grade;
+}
+
+const imageTable = {
+  SEED: Seed,
+  SPROUT: Sprout,
+  SMALL_TREE: YoungTree,
+  BIG_TREE: LargeTree,
+};
+
+export default function Aside({
```

Comment:
`Aside`는 너무 범용적인 컴포넌트 이름인 것 같아서 `Sidebar`를 제안하고 싶어요.

```suggestion
export default function Sidebar({
```

Reply:
사이드바가 더 좋겠네요! 추천감사합니다!